app/frontend/src/components/DashboardBanners/DashboardBanners.tsx

```diff
@@ -0,0 +1,72 @@
+import { Link as MuiLink } from "@material-ui/core";
+import { Alert as MuiAlert } from "@material-ui/lab/";
+import Alert from "components/Alert";
+import { Error as GrpcError } from "grpc-web";
+import { GetAccountInfoRes } from "pb/account_pb";
+import { accountInfoQueryKey } from "queryKeys";
+import React from "react";
+import { useQuery } from "react-query";
+import { Link } from "react-router-dom";
+import { editProfileRoute, settingsRoute } from "routes";
+import { service } from "service";
+
+import {
+  CLICK_HERE_TO_EIDT,
+  DONT_YOU_HATE,
+  FILL_IN_WHO_I_AM,
+  PASSWORD_TEXT_1,
+  PASSWORD_TEXT_2,
+  PASSWORD_TEXT_LINK,
+  PLEASE_COMPLETE_PROFILE,
+  UPLOAD_PHOTO,
+} from "./constants";
+
```

Comment:
Hmm I think it might be better if this file is in a folder called `features/home` or `feature/dashboard` instead, as I'm not sure it's reusable elsewhere? We try to put things in `components` when they are generic ones that can be reused across the app.